spec/unit/anima/attribute_spec.rb

```diff
@@ -47,8 +47,8 @@ def initialize(foo)
     context 'when attribute hash does not contain key' do
       let(:attribute_hash) { {} }
 
-      it 'should raise error' do
-        expect { subject }.to raise_error(Anima::Error::Missing, Anima::Error::Missing.new(target, :foo).message)
+      it 'should return without setting nil value' do
+        expect(subject).to eq nil
```

Comment:
If you follow my suggestion from https://github.com/mbj/anima/pull/16/files#r36578909, we should expect a KeyError here.